### ISSUE TITLE
"noarch: python" subpackages on conda-forge

### ISSUE THREAD

**Issue by btjanaka** [conda-forge/conda-forge.github.io#1906]
### Your question:

Hi! I am wondering how to set up subpackages which are noarch python. I am working on a library called pyribs. On PyPI, we have a few distributions with extras: ribs, ribs\[visualize\], and ribs\[all\]. On Conda, I would like to make these pyribs, pyribs-visualize, and pyribs-all, respectively.

I've successfully set up the packages with [this recipe](https://github.com/conda-forge/pyribs-feedstock/blob/main/recipe/meta.yaml) based on [this recipe from Gymnasium](https://github.com/conda-forge/gymnasium-feedstock/blob/main/recipe/meta.yaml). However, my packages all show up as requiring Python 3.10, and each one is only built for linux-64 as seen [here](https://anaconda.org/conda-forge/pyribs/files).

In short, I'm confused by two things:

1. To make the package noarch, I've put `noarch: python` under the `build` section of the main package, yet the subpackages show up as linux-64.
2. Even though I put my runtime requirements as Python >= 3.7 in my recipe, all the output packages end up depending on Python 3.10.

What is the correct way to make all the packages show up as noarch with Python >= 3.7 as a dependency?

**Comment by carterbox** [comment]
Mark each output as noarch individually.

**Comment by carterbox** [comment]
Metapackages do not need host requirements. Use `noarch: generic` for them.

**Comment by btjanaka** [comment]
Thank you @carterbox! That worked. For future reference, here was the PR where I made the change: https://github.com/conda-forge/pyribs-feedstock/pull/7
